apps/tangle-dapp/containers/Layout/Layout.tsx

```diff
@@ -37,6 +38,8 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
         <Footer isMinimal className="py-8" />
       </main>
 
+      <TxnConfirmationContainer />
```

Comment:
I think `TxnConfirmationModalContainer` is a better name to let others know this is a modal